app/src/androidTest/java/com/bottlerocketstudios/brarchitecture/HomeScreenUITest.kt

```diff
@@ -0,0 +1,115 @@
+package com.bottlerocketstudios.brarchitecture
+
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onRoot
+import com.bottlerocketstudios.brarchitecture.domain.models.GitRepository
+import com.bottlerocketstudios.brarchitecture.domain.models.Link
+import com.bottlerocketstudios.brarchitecture.domain.models.Links
+import com.bottlerocketstudios.brarchitecture.domain.models.User
+import com.bottlerocketstudios.brarchitecture.domain.models.Workspace
+import com.bottlerocketstudios.compose.home.HomeScreen
+import com.bottlerocketstudios.compose.home.HomeScreenState
+import com.bottlerocketstudios.compose.home.UserRepositoryUiModel
+import com.bottlerocketstudios.compose.util.StringIdHelper
+import com.bottlerocketstudios.compose.util.asMutableState
+import com.karumi.shot.ScreenshotTest
+import org.junit.Rule
+import org.junit.Test
+import java.time.ZonedDateTime
+
+class HomeScreenUITest : ScreenshotTest {
+
+    @get:Rule
+    val composeRule = createComposeRule()
+
+    @Test
+    fun homeScreenWithListItemsScreenshot() {
+        renderComponent(
+            HomeScreenState(
+                repositories = listOf(testCard1, testCard2).asMutableState(),
+                itemSelected = {}
+            )
+        )
+
+        compareScreenshot(composeRule.onRoot())
+    }
+
+    @Test
+    fun homeScreenNoResultItemsScreenshot() {
+        renderComponent(
+            HomeScreenState(
+                repositories = emptyList<UserRepositoryUiModel>().asMutableState(),
+                itemSelected = {}
+            )
+        )
+
+        compareScreenshot(composeRule.onRoot())
+    }
+
+    fun renderComponent(state: HomeScreenState? = null) {
```

Comment:
@Br-DiorD Think I    figured out our issue.  We should wrap HomeScreen like this:

ArchitectureDemoTheme {
     HomeScreen(state)
}

Also, Could HomeScreenState in function signature be non null?

Reply:
@BottleRocket-Colin nice! I'll give it a shot after this meeting. I do wonder why it is only an issue on certain screenshots because the Pull RequestScreenshot shot is fine.

Reply:
@Br-DiorD _ would say that the PR request screenshot is probably not correct but just closer to being correct.  It's a matter of where our h4 and h5 typographies are closer to the default sizes but our h1 is very small compared to the h1 default.  So some screens will show closer to correct than others depending on which typography elements are being used.

Reply:
@BottleRocket-Colin fixed. Sorry for the delay :)